packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -30,7 +30,7 @@ const MyProfile = () => {
     return (
         <AutoSizer>
             {({ height, width }) => (
-                <div className='my-profile' height={height} style={{ width }}>
```

Comment:
The height was not automatically adjusted since it was not passed to styling (this caused so much debugging pain 😭 )

Reply:
Or is this expected, I hope not